examples/models/transform_jitter.py

```diff
@@ -15,7 +15,7 @@
 normal = Jitter(width=0.2, distribution="normal")
 uniform = Jitter(width=0.2, distribution="uniform")
 
-p = figure(x_range=(0, 4), y_range=(0,10))
+p = figure(x_range=(0, 4), y_range=(0,10),toolbar_location='right')
```

Comment:
@birdsarah I added that because otherwise the toolbar was interfering with the labels (original, normal, uniform). I first started moving the labels around but then realized moving the toolbar was the more elegant solution!